doxia-doc-renderer/src/main/javadoc/org/apache/maven/doxia/docrenderer/itext/package.html

```diff
@@ -20,7 +20,7 @@
 -->
 
 <body>
-  <h1><b>DEPRECATED</b>.</h1>
+  <b>DEPRECATED</b>.
```

Comment:
Drop this hunk, it has already been removed in an open PR.

Reply:
I know, I can wait for merge your PR